video_worker/api_communicate.py

```diff
@@ -210,8 +218,8 @@ def send_val_data(self):
             )
 
             if r2.status_code > 299:
+                logger.error('VAL POST/PUT Fail: VAL')
                 ErrorObject().print_error(
-                    method=self,
```

Comment:
@yro I didn't know why this is used but when passed this argument an unexpected kwarg error was being thrown by `ErrorObject().print_error`. I checked `ErrorObject().print_error` it also didn't had any impementation for `method` argument.